addons/xeh/stringtable.xml

```diff
@@ -3,6 +3,7 @@
     <Package name="XEH">
         <Key ID="STR_CBA_XEH_Component">
             <English>Community Base Addons - Extended Event Handlers</English>
+            <German>Community Base Addons - Erweiterte Ereignishandler</German>
```

Comment:
Ereignisbehandlung

Reply:
Apparently it's a toss up: https://www.dict.cc/english-german/event+handler.html

Reply:
I wouldn't translate this at all since it's a code thing. You didn't translate strings to "Zeichenketten" either.

Reply:
Just be consistent.

_https://help.sap.com/doc/abapdocu_752_index_htm/7.52/de-DE/abenevent_handler_glosry.htm_
_https://de.wikipedia.org/wiki/Ereignis_(Programmierung)_

Reply:
>Auch die Ereignisbehandlungsroutinen (englisch event handler, deutsch Ereignishandler) und die Ereignisse selbst werden dann als Objekte modelliert.

Reply:
when you want to keep it then please also translate String to "Zeichenketten"

Reply:
^ done